Bottom: 9a5ec672bda784c35e1f86d0f274cdb9e58f217c
Top:    d5ba9adc90be6728f409ed99bc36a2fc6e659db0
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-13 15:29:40 -0300

Thread safety documentation.

for ChangeLog

	* manual/math.texi: Document thread safety properties.


---

diff --git a/manual/math.texi b/manual/math.texi
index 193d415..fd5d0c8 100644
--- a/manual/math.texi
+++ b/manual/math.texi
@@ -560,6 +560,7 @@ negative), @code{logb} returns @math{@infinity{}}.  If @var{x} is zero,
 @comment math.h
 @comment ISO
 @deftypefunx int ilogbl (long double @var{x})
+@safety{@mtsafe{}@assafe{}@acsafe{}}
 These functions are equivalent to the corresponding @code{logb}
 functions except that they return signed integer values.
 @end deftypefun
